README.rst

```diff
@@ -28,7 +28,9 @@ The manual for Dwarf Therapist can be downloaded separately.
 Building
 ========
 
-Dwarf-Therapist requires a C++ compiler (with C++14 support), cmake (3.1.0 or newer), and Qt5 (with Widgets and QML modules).
+Dwarf-Therapist requires a C++ compiler (with C++14 support), cmake (3.1.0 or newer), and Qt5.5 (with Widgets and QML modules).
+
+For All Users:
```

Comment:
I think it would be better with some kind of formatting, to make the distinction between sections (all/linux/mac) clearer: bold or low level title.

Reply:
Yep, I can poke.